packages/busybox.rb

```diff
@@ -14,1200 +14,1205 @@ class Busybox < Package
+    CONFIG_LFS=y
+    # CONFIG_PAM is not set
+    CONFIG_FEATURE_DEVPTS=y
+    # CONFIG_FEATURE_UTMP is not set
+    # CONFIG_FEATURE_WTMP is not set
+    # CONFIG_FEATURE_PIDFILE is not set
+    CONFIG_PID_FILE_PATH="/usr/local/var/run"
+    CONFIG_BUSYBOX=y
+    CONFIG_FEATURE_SHOW_SCRIPT=y
+    CONFIG_FEATURE_INSTALLER=y
+    CONFIG_INSTALL_NO_USR=y
+    # CONFIG_FEATURE_SUID is not set
+    # CONFIG_FEATURE_SUID_CONFIG is not set
+    # CONFIG_FEATURE_SUID_CONFIG_QUIET is not set
+    # CONFIG_FEATURE_PREFER_APPLETS is not set
+    CONFIG_BUSYBOX_EXEC_PATH="/proc/self/exe"
+    # CONFIG_SELINUX is not set
+    # CONFIG_FEATURE_CLEAN_UP is not set
+    CONFIG_FEATURE_SYSLOG_INFO=y
+    CONFIG_FEATURE_SYSLOG=y
 
-      #
-      # Build Options
-      #
-      # CONFIG_STATIC is not set
-      # CONFIG_PIE is not set
-      # CONFIG_NOMMU is not set
-      CONFIG_BUILD_LIBBUSYBOX=y
-      # CONFIG_FEATURE_LIBBUSYBOX_STATIC is not set
-      CONFIG_FEATURE_INDIVIDUAL=y
-      CONFIG_FEATURE_SHARED_BUSYBOX=y
-      CONFIG_CROSS_COMPILER_PREFIX=""
-      CONFIG_SYSROOT=""
-      CONFIG_EXTRA_CFLAGS="-pipe -O2 -fuse-ld=gold -flto=auto -fomit-frame-pointer"
-      CONFIG_EXTRA_LDFLAGS="-flto=auto"
-      CONFIG_EXTRA_LDLIBS="pthread"
-      # CONFIG_USE_PORTABLE_CODE is not set
-      # CONFIG_STACK_OPTIMIZATION_386 is not set
-      # CONFIG_STATIC_LIBGCC is not set
+    #
+    # Build Options
+    #
+    # CONFIG_STATIC is not set
+    # CONFIG_PIE is not set
+    # CONFIG_NOMMU is not set
+    CONFIG_BUILD_LIBBUSYBOX=y
+    # CONFIG_FEATURE_LIBBUSYBOX_STATIC is not set
+    CONFIG_FEATURE_INDIVIDUAL=y
+    CONFIG_FEATURE_SHARED_BUSYBOX=y
+    CONFIG_CROSS_COMPILER_PREFIX=""
+    CONFIG_SYSROOT=""
+    CONFIG_EXTRA_CFLAGS="-Os -pipe -flto -ffat-lto-objects -fPIC -fuse-ld=gold -pthread"
+    CONFIG_EXTRA_LDFLAGS=""
+    CONFIG_EXTRA_LDLIBS="pthread"
+    # CONFIG_USE_PORTABLE_CODE is not set
+    # CONFIG_STACK_OPTIMIZATION_386 is not set
+    # CONFIG_STATIC_LIBGCC is not set
 
-      #
-      # Installation Options ("make install" behavior)
-      #
-      CONFIG_INSTALL_APPLET_SYMLINKS=y
-      # CONFIG_INSTALL_APPLET_HARDLINKS is not set
-      # CONFIG_INSTALL_APPLET_SCRIPT_WRAPPERS is not set
-      # CONFIG_INSTALL_APPLET_DONT is not set
-      # CONFIG_INSTALL_SH_APPLET_SYMLINK is not set
-      # CONFIG_INSTALL_SH_APPLET_HARDLINK is not set
-      # CONFIG_INSTALL_SH_APPLET_SCRIPT_WRAPPER is not set
-      CONFIG_PREFIX="#{CREW_DEST_PREFIX}"
+    #
+    # Installation Options ("make install" behavior)
+    #
+    CONFIG_INSTALL_APPLET_SYMLINKS=y
+    # CONFIG_INSTALL_APPLET_HARDLINKS is not set
+    # CONFIG_INSTALL_APPLET_SCRIPT_WRAPPERS is not set
+    # CONFIG_INSTALL_APPLET_DONT is not set
+    # CONFIG_INSTALL_SH_APPLET_SYMLINK is not set
+    # CONFIG_INSTALL_SH_APPLET_HARDLINK is not set
+    # CONFIG_INSTALL_SH_APPLET_SCRIPT_WRAPPER is not set
+    CONFIG_PREFIX="/usr/local/tmp/crew/dest/usr/local"
```

Comment:
This should be abstracted with `#{CREW_PREFIX}` in place of `/usr/local/tmp/crew/dest/usr/local`.